README.adoc

```diff
@@ -28,6 +28,9 @@ Add the upstream wb projects too, because you might want to make changes on thos
 
 * Git clone: `guvnor`, `kie-wb-common`, `drools-wb` too (besides `optaplanner-wb`)
 * Import them all in the *same* IntelliJ project
+* Make sure to use `Maven -> Reimport All Maven Projects` and `Maven -> Generate Sources and Update Folders for All Projects` to update module dependencies
+* Run `grep -r -h --include=beans.xml "exclude" .` in `guvnor`, `kie-wb-common`, `drools-wb` project root directories and paste the output to `optaplanner-wb/optaplanner-wb-webapp/src/main/webapp/WEB-INF/beans.xml`
```

Comment:
@ge0ffrey Agreed. It might introduce minor maintenance burden as we'll have to keep the file up-to-date when new modules are added, however it's definitely cleaner approach.